param/parameterized.py

```diff
@@ -864,6 +864,8 @@ def __set__(self,obj,val):
                 obj.__dict__[self._internal_name] = val
 
         self._post_setter(obj, val)
+        if hasattr(val, '_setup_dependencies'):
```

Comment:
Ideally would have a better condition here, but can't do `isinstance(val, Parameterized)` since this is evaluated before Parameterized exists.

Reply:
Makes sense.